### DISCUSSION TITLE
Show dependents by a specific version

### DISCUSSION THREAD

**Discussion by jimmywarting** [npm/feedback#603] [Website]
I'm seeing someone still depending on a very old version in one of my packages that was released waaaay back. But haven't updated to a newer version yet...

It would be interesting to see what packages depends on a specific version if you go into the "versions" tab and select one.

Take [npm-check](https://www.npmjs.com/package/npm-check) as an example... it shows 212 dependents overall
but if i select [npm-check@5.7.1](https://www.npmjs.com/package/npm-check/v/5.7.1) specifically that current have 171 weekly download of something that was released 4 years ago, then it still shows 212 dependents overall. I wish it could narrow it down.

![image](https://user-images.githubusercontent.com/1148376/142740481-2f4f7e79-2cd1-4913-8390-f038acbac303.png)

**Comment by targos** [comment]
It was possible in the past using a CouchDB view. I even made a package for it (https://github.com/targos/npm-depends).
Unfortunately, the view was removed a couple years ago.

**Comment by MylesBorins** [comment]
@jimmywarting This is definitely something interesting and worth exploring. I'm going to leave this discussion unanswered to revisit in the new year. Thanks for the suggestion.
